psi4/CMakeLists.txt

```diff
@@ -187,8 +197,6 @@ install(PROGRAMS ${CMAKE_BINARY_DIR}/psi4
         DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR})
 
     # <<<  install lib/  >>>
-file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}${PYMOD_INSTALL_LIBDIR}/psi4)
```

Comment:
Have you tried a build with a non-default `PYMOD_INSTALL_LIBDIR`? If not, I'll exercise it in the course of a trial conda build. It may be that this line needs modifying instead of dropping.

Reply:
No, I have not tried!

Reply:
conda build (that uses non-default `PYMOD_INSTALL_LIBDIR`) worked fine, so I guess this is safe.